xedocs/xedocs.py

```diff
@@ -127,6 +127,19 @@ def xedocs_protocol(name, version="ONLINE", sort=None, attr=None, **kwargs):
             return docs[0]
 
         return docs
+    
+    @URLConfig.register('docs-to-dict')
```

Comment:
can be used for any list of objects. 
```suggestion
    @URLConfig.register('objects-to-dict')
```